src/plugins/data/public/search/session/session_service.ts

```diff
@@ -567,14 +567,15 @@ export class SessionService {
     if (this.isCurrentSession(sessionId)) {
       this.state.transitions.store(searchSessionSavedObject);
 
-      // trigger new poll for all completed searches that are not stored to propogate them into newly creates search session saved object and extend their keepAlive
-      const completedSearches = this.state
+      // trigger a poll for all the searches that are not yet stored to propagate them into newly created search session saved object and extend their keepAlive
+      const searchesToExtend = this.state
         .get()
         .trackedSearches.filter(
-          (s) => s.state === TrackedSearchState.Completed && !s.searchMeta.isStored
```

Comment:
This code is executed when the user saves a session. Previously after the user has saved the session we polled only completed searches, expecting that in-progress searches will be saved in the search saved object on the next interval loop in ~1 second. But now with the interval increase, we will also poll in-progress searches separately here.